dashboards/apache-traffic-server/apache-traffic-server.json

```diff
@@ -0,0 +1,1583 @@
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.area"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"legend": {
+							"enabled": true
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT latest(global.proxy.process.http.response_document_size_100) AS 'size 100', latest(global.proxy.process.http.response_document_size_1K) AS 'size 1K',latest(global.proxy.process.http.response_document_size_5K) AS 'size 5K', latest(global.proxy.process.http.response_document_size_10K)AS 'size 10K',latest(global.proxy.process.http.response_document_size_1M)AS 'size 1M', latest(global.proxy.process.http.response_document_size_inf)AS 'size inf'  FROM ATSHttpSampleMetrics TIMESERIES AUTO "
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "HTTP Request Document Size (bytes)",
+					"layout": {
+						"column": 9,
+						"row": 4,
+						"width": 4,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.line"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"legend": {
+							"enabled": true
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT latest(global.proxy.process.http.request_document_size_100) AS 'size 100', latest(global.proxy.process.http.request_document_size_1K)AS 'size 1K', latest(global.proxy.process.http.request_document_size_3K) AS 'size 3K', latest(global.proxy.process.http.request_document_size_5K) AS 'size 5K', latest(global.proxy.process.http.request_document_size_10K)AS 'size 10K', latest(global.proxy.process.http.request_document_size_1M) AS 'size 1M', latest(global.proxy.process.http.request_document_size_inf)AS 'size inf' FROM ATSHttpSampleMetrics TIMESERIES AUTO "
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						},
+						"yAxisLeft": {
+							"zero": true
+						}
+					}
+				},
+				{
+					"title": "HTTP Total Incoming Statistics",
+					"layout": {
+						"column": 1,
+						"row": 7,
+						"width": 4,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.area"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"legend": {
+							"enabled": true
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT latest(global.proxy.process.http.total_incoming_connections) AS 'incoming connections', latest(global.proxy.process.http.incoming_responses) AS 'incoming responses' FROM ATSHttpSampleMetrics TIMESERIES AUTO "
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "HTTP Average Transactions Per Client & Server",
```

Comment:
```suggestion
					"title": "HTTP Average Transactions per Client & Server",
```

Reply:
Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.